go/libkb/sync_secret.go

```diff
@@ -63,6 +63,9 @@ type SecretSyncer struct {
 	Contextified
 	dirty bool
 	keys  *ServerPrivateKeys
+	// forceNextSync makes it so next time syncFromServer is performed,
+	// version will be disregarded and latest keys will always be fetched.
+	forceNextSync bool
```

Comment:
not a huge fan of this, is there any reason we can't just change `RunSyncer` and push the force mode all the way down?